Formula/rtv.rb

```diff
@@ -18,7 +18,7 @@ class Rtv < Formula
   depends_on :python3
 
   def install
-    venv = virtualenv_create(libexec, "python3")
+    venv = virtualenv_create(libexec, "python")
```

Comment:
that needs to be python3.

Reply:
It wouldn't compile locally for me with python3. Happy to change it, but I can't promise that it'll work.